Library/include/CSP/Multiplayer/SpaceEntity.h

```diff
@@ -341,6 +343,7 @@ class CSP_API SpaceEntity
 	bool IsTransferable;
 	bool IsPersistant;
 	uint64_t OwnerId;
+	uint64_t ParentId;
```

Comment:
Observation - we are serialising null for no-parent, but representing in CSP as 0.
Suggestion - Should we align those two things? Perhaps by using csp::Optional<uint64_t> for the type of `ParentId` and using null when there's no parent?

Reply:
I'm happy to do that

Reply:
Updated